<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorDemo</class>
 <widget class="QWidget" name="SensorDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SensorDemo</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>211</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="open_port">
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="close_port">
      <property name="text">
       <string>关闭串口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>端口配置</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>端口号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="com"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>波特率：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="baud">
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14400</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>数据位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="databit">
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>校验位：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="checkbit">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Odd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Even</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>停止位：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="stopbit">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>接收窗口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="Receive_text_win">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QComboBox" name="receive_state">
        <item>
         <property name="text">
          <string>hex</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>string</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear_receive_button">
        <property name="text">
         <string>清空数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>Receive_text_win</zorder>
   <zorder>open_port</zorder>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>close_port</zorder>
  </widget>
  <widget class="QLabel" name="sensor_data">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>140</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(233, 185, 110);</string>
   </property>
   <property name="text">
    <string>超声波雷达</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>553</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(31, 238, 205);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="distance">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>距离(mm):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>611</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(242, 24, 24);</string>
   </property>
   <property name="text">
    <string>说明：
1.报警距离设置的范围为：240~9999，在该距离内发生报警。
2.超声波测距可提供 230mm-6000mm 的非接触式距离感测功能，测距精度可达 3mm；
3.测距时，被测物体的面积不少于 0.5 平方米且平面尽量要求平整，否则影响测量的结果。</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>260</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>240</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
  </widget>
  <widget class="QLabel" name="message">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>260</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="distance_data">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>报警距离(mm):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>420</y>
     <width>89</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <zorder>label_9</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>sensor_data</zorder>
  <zorder>label_2</zorder>
  <zorder>distance</zorder>
  <zorder>label_10</zorder>
  <zorder>spinBox</zorder>
  <zorder>message</zorder>
  <zorder>distance_data</zorder>
  <zorder>quit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
